service/src/main/java/bio/terra/profile/service/profile/flight/delete/UnlinkBillingProfileIdFromMrgStep.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.profile.service.profile.flight.delete;
+
+import bio.terra.profile.service.azure.ApplicationService;
+import bio.terra.profile.service.profile.model.BillingProfile;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.stairway.exception.RetryException;
+
+/** Removes the tag that associates a billing profile ID with an azure managed resource group */
+public record UnlinkBillingProfileIdFromMrgStep(
+    ApplicationService applicationService, BillingProfile profile) implements Step {
+  private static final String BILLING_PROFILE_ID_TAG = "terra.billingProfileId";
```

Comment:
This constant is also in `LinkBillingProfileIdFromMrgStep`, so it should probably be located somewhere else. I'm not sure where the best place to put it is, in this project.

Reply:
I suggest a common constants class.